skyrim-platform/src/platform_se/CMakeLists.txt

```diff
@@ -168,11 +168,15 @@ if(NOT "${SKIP_SKYRIM_PLATFORM_BUILDING}")
   add_custom_target(RestartGame ALL COMMAND ${CMAKE_COMMAND} -E touch ${CMAKE_CURRENT_BINARY_DIR}/touch_$<CONFIG>)
   add_dependencies(RestartGame ${DEPENDENCIES_FOR_CUSTOM_TARGETS})
 
-  add_custom_target(codegen ALL
-    COMMAND node ${PROJECT_SOURCE_DIR}/src/platform_se/codegen/index.js
-    WORKING_DIRECTORY ${wdir}
-  )
-  list(APPEND DEPENDENCIES_FOR_CUSTOM_TARGETS codegen)
+  # add_custom_target(codegen ALL
```

Comment:
These lines are still required to run codegen during project building.
Make sure that codegen runs after its compilation (use `add_dependencies`)

See:
https://cmake.org/cmake/help/latest/command/add_dependencies.html
https://cmake.org/cmake/help/latest/command/add_custom_target.html
https://cmake.org/cmake/help/latest/manual/cmake-generator-expressions.7.html#genex:TARGET_FILE